Main.js

```diff
@@ -1488,6 +1537,23 @@ function init_ui() {
 
 
 	init_journal($("#message-broker-client").attr("data-gameId"));
+	
+	if (window.DM) {
+		// use DDB character tools to update character info every 10 seconds
+		loadScript("https://media.dndbeyond.com/character-tools/vendors~characterTools.bundle.f8b53c07d1796f1d29cb.min.js",

```

Comment:
just to be clear, the only reason why you defer this load is because you need to wait to see if the current user is the DM, right?

Reply:
Yes, I didn't want to load that large and busy javascript file unless it's needed by the DM